terser.md

```diff
@@ -9,7 +9,7 @@ and in console
 
 ```
 cd src
-terser ./src/o.js -c -m toplevel,reserved=["o","element","obj","name","val","attributes", "attrs", "classNames", "classList", "className", "id", "children", "text", "html", "oRefInstance", "parentNode", "childNode"] -o "./npm/o.js"
+terser ./src/o.js -c -m toplevel,reserved=["o","element","obj","name","val","attributes", "attrs", "classNames", "classList", "className", "id", "children", "text", "html", "oRefInstance", "parentNode", "childNode", "oRender", "oRef", "cleanParentContent"] -o "./npm/o.js"
```

Comment:
Proponuję spróbować to spakować parcelem zamiast ręcznie terserem robić. Przy oComponents zgrabnie mi to wyszło. Przetestujmy czy nie będzie wydajniej tym parcelem. Chętnie się tym zajmę